src/PyCall.jl

```diff
@@ -286,6 +286,20 @@ end
 
 getindex(o::PyObject, s::Symbol) = convert(PyAny, getindex(o, string(s)))
 
+function getproperty(o::PyObject, s::Symbol)
```

Comment:
Note that I think we also want to accept an `AbstractString` here so that you can do `o."foo"` and get the raw `PyObject`, similar to the current `o["foo"]`.